src/components/theme/View/NewsItemView.jsx

```diff
@@ -41,15 +38,12 @@ const NewsItemView = ({ content }) => {
       )}
       {content.image && (
         <Image
-          className="documentImage"
+          className="documentImage ui right floated image"
           alt={content.title}
           title={content.title}
-          src={
-            content.image['content-type'] === 'image/svg+xml'
```

Comment:
Does the new component need special handling for SVGs?

Reply:
Actually, probably it does. Adding it

Reply:
Done